js_modules/dagster-ui/packages/ui-core/src/graph/OpTags.tsx

```diff
@@ -378,7 +378,7 @@ function generateColorForLabel(label = '') {
     label
       .split('')
       .map((c) => c.charCodeAt(0))
-      .reduce((n, a) => n + a) % 360
+      .reduce((n, a) => n + a, 0) % 360
```

Comment:
Bah... I've actually been hit by this one before, I really dislike that the initial value is optional 🤦‍♂️ I googled around to see if there's an ESLint rule that will force the initial value but it doesn't seem like it. Went through all the other `.reduce(` occurrences in the project and it does seem like this was the only one...